addons/website/static/src/systray_items/website_switcher.js

```diff
@@ -17,7 +18,7 @@ export class WebsiteSwitcherSystray extends Component {
             name: website.name,
             id: website.id,
             callback: () => {
-                if (website.domain && website.domain !== window.location.origin) {
+                if (website.domain && !wUtils.isHTTPSorNakedDomainRedirection(website.domain, window.location.origin)) {
```

Comment:
Just to understand: in this case, it's not critically necessary, right (no infinite loop)? You are just on `hello.world` with a button which would redirect you to `www.hello.world` needlessly if choosing to click on the same website you are already on? (but of course even if not critical better to fix as a one line fix :+1: )